File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        ArrayList<Integer> temp=new ArrayList();
        int c=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]!=0){
                temp.add(nums[i]);
                c++;
            }
        }
        for(int i=0;i<c;i++){
            nums[i]=temp.get(i);
        }
        for(int i=c;i<nums.length;i++){
             nums[i]=0;
        }        
    }
}

File: 0283-move-zeroes/0283-move-zeroes.cs
public class Solution {
    public void MoveZeroes(int[] nums) {
        int c=0,zero=0;
        for(int i=0;i<nums.Length;i++){
            if(nums[i]!=0){
                nums[c++]=nums[i];
            }
            else{
                zero++;
            }
        }
        for(int i=nums.Length-1;i>nums.Length-1-zero;i--){
            nums[i]=0;
        }
    }
}